MBBSEmu.Tests/CPU/INC_Tests.cs

```diff
@@ -0,0 +1,178 @@
+﻿using Iced.Intel;
+using MBBSEmu.CPU;
+using MBBSEmu.Extensions;
+using Xunit;
+using static Iced.Intel.AssemblerRegisters;
+
+namespace MBBSEmu.Tests.CPU
+{
+    public class INC_Tests : CpuTestBase
+    {
+        [Theory]
+        [InlineData(0,1, false, false, false)]
+        [InlineData(0xFF, 0, true, false, false)]
+        [InlineData(0x7F, 0x80, false, true, true)]
+        [InlineData(0x80, 0x81, false, false, true)]
+        public void INC_R8(byte alValue, byte expectedResult, bool zeroFlagSet, bool overflowFlagSet, bool signFlagSet)
```

Comment:
I feel like a lot of these tests could be more concise if instead of passing 3-4 flag values in the `InlineData` areas, you just pass a single expected `EnumFlags` with a combination of all the expected things.